src/components/metadata/metadata.js

```diff
@@ -166,6 +226,20 @@ const MetaData = ({
                   label="Type:"
                   value={translateMetadataType(metadata.type)}
                 />
+                <MetaDataRow
+                  label="Error Log:"
+                  visible={isWorkflowView && hasError}
+                >
+                  <ErrorLog
+                    errorDetails={currentErrorDetails}
+                    className="pipeline-metadata__error-log"
+                    onToggleCode={handleErrorLogToggle}
+                    dataTest={getDataTestAttribute('metadata', 'error-log')}
```

Comment:
I just noticed this. It's a good change, but I noticed we don't do this everywhere so pls let's create a follow up ticket to apply this across codebase. @jitu5 ?